test/types/TestChangeMode.cpp

```diff
@@ -0,0 +1,79 @@
+#include "../../include/types/Perf.h"
+#include "../../include/types/PerformanceTable.h"
+#include "../../include/utils.h"
+
+#include "gtest/gtest.h"
+#include <sstream>
+#include <utility>
+
+// Checking PerformanceTable changeModes
+
+TEST(TestPerformanceTable, TestChangeMode1) {
```

Comment:
If you don't execute any test in here, best remove it :)

Reply:
Yup didn't see I added it.